extension-ui/oops.html

```diff
@@ -17,4 +17,27 @@ <h1 id="header" class="header"></h1>
     </div>
   </div>
 </body>
+</html> -->
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Access Blocked</title>
+  <!-- קישור לסטיילים אם יש לך צורך בסטיילים נוספים -->
+  <link rel="stylesheet" href="oops.css">
+  <!-- קישור לקובץ ה-JavaScript שנוצר על ידי Webpack -->
+  <script src="bundle.js" defer></script>
+</head>
+<body class="container">
+  <div>
+    <div class="content">
+      <h1 id="header" class="header"></h1>
+      <p id="paragraph" class="paragraph"></p>
+    </div>
+    <div class="btn-class">
+      <button id="btn" class="btn"></button>
```

Comment:
1. please remove all Hebrew comments, or replace them with English
2. replace button with our GenericButton, check that it works